packages/react-icons/convert.js

```diff
@@ -149,13 +121,11 @@ export const ${destFilename} = /*#__PURE__*/wrapIcon(/*#__PURE__*/${destFilename
   /** @type string[][] */
   const iconChunks = [];
   while(iconExports.length > 0) {
-    iconChunks.push(iconExports.splice(0, 500));
```

Comment:
The file size is much smaller now, put more icon into one file will be a little faster.